library/src/main/java/com/novoda/downloadmanager/MigrationServiceBinder.java

```diff
@@ -0,0 +1,52 @@
+package com.novoda.downloadmanager;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.IBinder;
+
+public class MigrationServiceBinder {
```

Comment:
Handles the binding without exposing `ServiceConnection` to the client applications of the library.

Reply:
This will be iterated upon to refine the definition of `bound`